pages/entry.jsx

```diff
@@ -50,16 +51,23 @@ export default React.createClass({
     });
   },
   render() {
+    let docTitle;
     let justPostedByUserMessage;
     let projectCard;
 
     if (this.state.dataLoaded) {
+      docTitle = `${this.state.entry.title}`;
       justPostedByUserMessage = this.state.justPostedByUser ? (<h5 className="text-center">Thanks for submitting!</h5>) : null;
       projectCard = this.state.dataLoaded ? <ProjectCard {...Utility.processEntryData(this.state.entry)} onDetailView={true} /> : null;
     }
 
     return (
       <div>
+        { docTitle &&
```

Comment:
this is basically an anti-pattern, where you're triggering core functionality as a side effect rather than clear intent.

Let's instead do something like:
```
let docTitle = null;

if (...) {
  docTitle = ...
  ...
}
if (doctitle) {
  docTitle = <Helmet><title>{ docTitle }</title></Helmet>;
}
return (
  <div>
  {docTitle}
  { just....
```